src/sentry/incidents/subscription_processor.py

```diff
@@ -322,24 +326,12 @@ def get_crash_rate_alert_metrics_aggregation_value(
         count is just ignored
         - `crashed` represents the total sessions or user counts that crashed.
         """
-        row = subscription_update["values"]["data"][0]
```

Comment:
We can just pull this function out entirely instead of pulling out parts of it

Reply:
I did pull out all of it besides `self.reset_trigger_counts()`, which can't be called outside the subscription processor.

Reply:
Chatted in slack: resetting trigger counts isn't used anywhere, so we can refactor the whole method.